src/CodeGen_Hexagon.h

```diff
@@ -24,8 +24,8 @@ class CodeGen_Hexagon : public CodeGen_Posix {
 
     void init_module();
 
-    void push_buffer(const std::string &name, int dimensions, llvm::Value *buffer);
-    void pop_buffer(const std::string &name, int dimensions);
+    void push_buffer(const std::string &name, int dimensions, llvm::Value *buffer, bool global = false);
```

Comment:
The previous definition of these two methods were overrides of virtual methods in CodeGen_LLVM; now they aren't. Does that matter?